src/components/toolbar/ToolBar.vue

```diff
@@ -4,9 +4,13 @@
       aria-label="Toolbar" :class="{ 'ignore-pointer': canvasDragPosition }"
     >
       <breadcrumb :breadcrumb-data="breadcrumbData" />
-      <div class="mr-3">
-        <align-buttons @save-state="$emit('save-state')" />
+      <div class="mr-3" style="display: flex;">
```

Comment:
There should be a class coming from Bootstrap to apply display flex here. Avoid inline styling